src/dev.sh

```diff
@@ -82,6 +82,8 @@ function rundotnet ()
 
 function generateConstant()
 {
+    # copy generate file template, so we won't get complie error
```

Comment:
If it's possible to add a comment that explains what is going on here, would be great.